solana-programs/audius_eth_registry/src/processor.rs

```diff
@@ -41,12 +40,45 @@ impl Processor {
         expected_signer: [u8; SecpSignatureOffsets::ETH_ADDRESS_SIZE],
         message: &[u8],
         secp_instruction_data: Vec<u8>,
+        instruction_index: u8,
     ) -> Result<(), AudiusError> {
-        let eth_address_offset = 12;
+        // Only single recovery expected per instruction
```

Comment:
Total nit, but ended up changing this in rewards manager - we're doing a bunch of useless addition to get the same fixed values here

Reply:
good call